### DISCUSSION TITLE
Videos: Unknown format, thumbnail creation failed

### DISCUSSION THREAD

**Discussion by ameralzarouni** [photoprism/photoprism#1324] [Q&A] [answered]
```
2021-05-23 08:31:41 WARN index: failed creating thumbnails for IMG_0621.MOV (image: unknown format)
2021-05-23 08:31:41 ERRO media: image: unknown format in IMG_0621.MOV.jpg
2021-05-23 08:31:41 INFO converting hdd2/100APPLE/IMG_0621.MOV to jpg
```
Any idea what's happening here?

**Comment by lastzero** [comment, marked as answer]
Seems a JPEG could not be properly created, maybe the MOV is broken or has an uncommon encoding FFmpeg can not read.

**Comment by lastzero** [comment]
May may use exiftool to view metadata including the actual file type to debug this.

**Comment by hiro5id** [comment]
I had similar issues earlier today.   I was getting errors like this:
```
level=warning msg="index: failed converting test_copy_of_PICT0010.43c3e016_testing.mov to jpeg (file type mov not supported)"
```

It turns out I had the option `Disable FFmpeg` checked under Advanced Options in the Web UI.

When I unchecked it, and restarted the server, it started working again.

I don't know for sure if that's what fixed it for me, may have been just coincidence, but perhaps check that setting in yours.

**Reply by lastzero** [reply]
FFmpeg and Exiftool must be enabled for videos. See *Troubleshooting Checklists*:

- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures
- https://docs.photoprism.app/getting-started/troubleshooting/#broken-thumbnails

> If some pictures have broken or missing thumbnails, first [check Library > Errors for errors and warnings](https://docs.photoprism.app/getting-started/troubleshooting/logs/). In case the application logs don't contain anything helpful:
> - [ ] FFmpeg and/or RAW converters are disabled in Settings > Advanced (enable them)
> ...
